### PR TITLE
Docker compose for Quickbooks Online

### PR DESCRIPTION
**Pull request by KirtiGautam**
Written docker compose file for Quickbooks and fixed the automatic restart of API when docker daemon starts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,60 +4,76 @@ Django Rest Framework API for Fyle Quickbooks Online Integration
...
+      FYLE_CLIENT_ID:
+      FYLE_CLIENT_SECRET:
+      FYLE_TOKEN_URI:
+      FYLE_JOBS_URL:
+      QBO_CLIENT_ID:
+      QBO_CLIENT_SECRET:
+      QBO_REDIRECT_URI:
+      QBO_TOKEN_URI:
+      QBO_ENVIRONMENT:
+      SERVICE_NAME:
```
Comment (ashwin1111) [inline README.md:36]: this is not needed

File: docker-compose.yml.template [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      FYLE_CLIENT_ID:
+      FYLE_CLIENT_SECRET:
+      FYLE_TOKEN_URI:
+      FYLE_JOBS_URL:
+      QBO_CLIENT_ID:
+      QBO_CLIENT_SECRET:
+      QBO_REDIRECT_URI:
+      QBO_TOKEN_URI:
+      QBO_ENVIRONMENT:
+      SERVICE_NAME:
```
Comment (ashwin1111) [inline docker-compose.yml.template:30]: same

File: docker-compose.yml.template [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
```
Comment (ashwin1111) [inline docker-compose.yml.template:58]: same

**Review by ashwin1111 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KirtiGautam) [reply README.md:36]: Okay, I will retract this

File: docker-compose.yml.template (hunk 2 above)
Comment (KirtiGautam) [reply docker-compose.yml.template:30]: And this.

File: docker-compose.yml.template (hunk 3 above)
Comment (KirtiGautam) [reply docker-compose.yml.template:58]: This too.

File: README.md [hunk 4]
```diff
@@ -4,60 +4,76 @@ Django Rest Framework API for Fyle Quickbooks Online Integration
...
 
     ```
-    pip install -r requirements.txt
+    docker-compose up -d db api qcluster
+    ```
+
+* The database can be accessed by this command, on password prompt type `postgres`
+
+    ```
+    docker-compose run db psql -h db -U postgres xero_db
```
Comment (ashwin1111) [inline README.md:54]: ```qbo_db```

Comment (KirtiGautam) [reply README.md:54]: Fixing the typo.

**Review by Shwetabhk — APPROVED** [review]

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**